LostAtlasCAH.API/LostAtlasCAH.API/obj/project.assets.json

```diff
@@ -0,0 +1,11901 @@
+{
```

Comment:
So this file seems to be where most of the difs are coming from. Idk why it keeps getting regenerated, might have to look into this.